azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurecr.go

```diff
@@ -47,14 +69,19 @@ func expandServiceEndpointAzureCR(d *schema.ResourceData) (*serviceendpoint.Serv
 			"tenantId":           d.Get("azurecr_spn_tenantid").(string),
 			"loginServer":        loginServer,
 			"scope":              scope,
+			"serviceprincipalid": d.Get("spn_id").(string),
```

Comment:
Since it's `computed`, why do we get its value?

Reply:
This parameters are required when we try to update the service connection